src/reducers/FridgeReducer.js

```diff
@@ -35,14 +35,20 @@ export const FridgeReducer = (state = initialState(), action) => {
       const { payload } = action;
       const { date } = payload;
 
-      return { ...state, fromDate: date };
+      const fromDate = new Date(date);
+      fromDate.setUTCHours(0, 0, 0, 0);
+
+      return { ...state, fromDate };
     }
 
     case FRIDGE_ACTIONS.CHANGE_TO_DATE: {
       const { payload } = action;
       const { date } = payload;
 
-      return { ...state, toDate: date };
+      const toDate = new Date(date);
+      toDate.setUTCHours(23, 59, 59, 999);
```

Comment:
Nit: I think incrementing the day and decrementing the ms is a bit clearer/tidier:

```suggestion
      toDate.setUTCDate(toDate.getUTCDate() + 1);
      toDate.setUTCHours(0, 0, 0, -1);
```

Could also do as a one-liner... but I think it's less clear than what you already have :laughing:.

```suggestion
      toDate.setUTCHours(24, 0, 0, -1);
```

Reply:
Maybe my lack of ARITHMETIC skills, but the 23,59,59,9999 is easier to grok then seeing the -1 and having to comprehend